charts/dave/charts/geodata-eai/templates/service.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "daveGeodataEai.fullname" . }}
+  labels:
+    {{- include "daveGeodataEai.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)